src/main/java/seedu/address/model/group/tasks/Deadline.java

```diff
@@ -38,39 +36,39 @@ public String getDeadline() {
     }
 
-    /**
-     * Returns a string representation of the `Deadline` task, including its description and deadline.
-     *
-     * @return A formatted string representing the `Deadline` task.
-     */
-    public String toString() {
-        // Format LocalDateTime as a string in your desired output format
-        DateTimeFormatter outputFormatter = DateTimeFormatter
-                .ofPattern("MMM dd yyyy h:mm a");
-        String formattedDateTime = by.format(outputFormatter);
-        return "[D]" + super.toString() + " (by: " + formattedDateTime + ")";
-    }
+//    /**
+//     * Returns a string representation of the `Deadline` task, including its description and deadline.
+//     *
+//     * @return A formatted string representing the `Deadline` task.
+//     */
+//    public String toString() {
+//        // Format LocalDateTime as a string in your desired output format
+//        DateTimeFormatter outputFormatter = DateTimeFormatter
+//                .ofPattern("MMM dd yyyy h:mm a");
+//        String formattedDateTime = by.format(outputFormatter);
+//        return "[D]" + super.toString() + " (by: " + formattedDateTime + ")";
+//    }
 
-    /**
-     * Formats the task's information into a user-friendly string representation.
-     * This method is used to generate a message indicating that a task has been added successfully.
-     *
-     * @return A formatted string message indicating the task addition.
-     */
-    public String printStr() {
-        return ("Got it. I've added this task:\n "
-                + this.toString() + "\n" + "Now you have "
-                + Task.getCounter() + " tasks in the list\n");
-    }
+//    /**
+//     * Formats the task's information into a user-friendly string representation.
+//     * This method is used to generate a message indicating that a task has been added successfully.
+//     *
+//     * @return A formatted string message indicating the task addition.
+//     */
+//    public String printStr() {
+//        return ("Got it. I've added this task:\n "
+//                + this.toString() + "\n" + "Now you have "
+//                + Task.getCounter() + " tasks in the list\n");
+//    }
 
-    /**
-     * Generates a string representation of the `Deadline` task for saving to a file.
-     *
-     * @return A formatted string representing the `Deadline` task for file storage.
-     */
-    @Override
-    public String generateStr() {
-        return "D | " + this.getStatus()
-            + " | " + this.getTask() + " | " + byStr + " | " + this.getModule();
-    }
+//    /**
+//     * Generates a string representation of the `Deadline` task for saving to a file.
+//     *
+//     * @return A formatted string representing the `Deadline` task for file storage.
+//     */
+//    @Override
+//    public String generateStr() {
+//        return "D | " + this.getStatus()
+//            + " | " + this.getTask() + " | " + byStr + " | " + this.getModule();
+//    }
```

Comment:
These parts are not needed?

Reply:
we dont use anywhere but jus commented out to be safe for now

Reply:
i think can delete it if we don't use.

Reply:
thanks! done!